src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -1173,4 +1166,43 @@ public boolean isStorageQuotasEnforced() {
     public Long getTestStorageQuotaLimit() {
         return settingsService.getValueForKeyAsLong(SettingsServiceBean.Key.StorageQuotaSizeInBytes);
     }
+
+    /*
+    RateLimitUtil will parse the json to create a List<RateLimitSetting>
+     */
+    public String getRateLimitsJson() {
+        return settingsService.getValueForKey(SettingsServiceBean.Key.RateLimitingCapacityByTierAndAction, "");
+    }
+
+    public Integer getIntFromCSVStringOrDefault(final SettingsServiceBean.Key settingKey, final Integer index, final Integer defaultValue) {
+        Integer value = defaultValue;
+        if (settingKey != null && !settingKey.equals("")) {
+            String csv = settingsService.getValueForKey(settingKey, "");
+            try {
```

Comment:
A minor thing, but this method is causing some unnecessary noise in the log when no tiers are defined (from the NumberFormatException below). 
Something like `if (!csv.equals(""))` here?

Reply:
fixed